Physics2D.Diagnostics/Diagnostics/DebugView.cs

```diff
@@ -143,6 +155,11 @@ private void DrawDebugData()
                     Transform xf = b.GetTransform();
                     foreach (Fixture f in b.FixtureList)
                     {
+                        if (EnableDebugFiltering)
```

Comment:
The contition for a shape to be visible is  (Flags & DebugViewFlags.Shape) != 0 
 && (!EnableDebugFiltering || _debuggableCategories.HasFlag(f.CollisionCategories))
 that's too many switch for a user to toggle. EnableDebugFiltering  is one too many switch since the user can set _debuggableCategories back to Category.All.